src/App.js

```diff
@@ -3,32 +3,35 @@ import Header from "./components/Header";
 import { createElement } from "./utils/elements";
 import { Character } from "./components/Character";
 import { Characters } from "./components/Characters";
-import { getCharacterById } from "./utils/api";
+import { getCharactersByName } from "./utils/api";
 
 function App() {
   const header = Header();
-  const characters = Characters();
+  const characterContainer = Characters();
   const main = createElement("main", {
     className: "main",
-    children: [characters],
+    children: [characterContainer],
   });
-
-  async function getCharacters() {
-    const firstCharacter = await getCharacterById(5);
-    const secondCharacter = await getCharacterById(2);
-    characters.append(
-      Character({
-        name: firstCharacter.name,
-        imgSrc: firstCharacter.image,
-      }),
+  async function loadCharacters(name) {
+    const namecharacters = await getCharactersByName(name);
+    const characterElements = namecharacters.map((character) =>
       Character({
-        name: secondCharacter.name,
-        imgSrc: secondCharacter.image,
+        name: character.name,
+        imgSrc: character.image,
       })
     );
+    characterContainer.innerHTML = "";
+    characterContainer.append(...characterElements);
   }
-  getCharacters();
```

Comment:
leave line 30 with` getCharacters()` to get pictures on inital siteload ;)

Reply:
Ah, thank you so much! I was wondering, what I was missing to have them there!